base_local_planner/src/speed_limiters/static_object_speed_limiter.cpp

```diff
@@ -99,15 +99,9 @@ namespace base_local_planner
       return true;
     }
 
-    if (const char *env_var = std::getenv("ROSEVN_STATIC_OBJECT_LIMITER_ENABLED"))
+    if (params_.enabled == false)
```

Comment:
this probably can be deleted entirely as the above if statement does effectively the same thing.